ValetTests/ValetTests.m

```diff
@@ -110,8 +110,11 @@ - (void)test_initialization_invalidArgumentsCauseFailure;
 {
     XCTAssertNil([[VALValet alloc] initWithIdentifier:@"" accessibility:VALAccessibilityAlways]);
     XCTAssertNil([[VALValet alloc] initWithIdentifier:@"test" accessibility:0]);
-    XCTAssertNil([[VALSynchronizableValet alloc] initWithIdentifier:@"test" accessibility:VALAccessibilityWhenPasscodeSetThisDeviceOnly]);
```

Comment:
Both of the below lines are `VALAccessibilityWhenUnlockedThisDeviceOnly`, should one be `VALAccessibilityWhenPasscodeSetThisDeviceOnly`?